projects/application/filehive.md

```diff
@@ -0,0 +1,65 @@
+## Project Name: Filehive
+Filehive
+
+## Category 
+Application, Wallet, Marketplace
+<!--developer tooling, application, wallet, infrastructure, etc-->
+
+## Project Description
+Data is the lifeblood of the Internet and therefore one of the most valuable digital assets that can be exchanged. Filehive is a new kind of marketplace built on top of Filecoin and IPFS to provide everyone across the planet an easy and cheap way to distribute and consume diverse datasets.
+
+The OB1 team has been working with Protocol Labs for many years now and this is the first Filecoin first project to come out of their development shop.
+
+<!--Describe your project in a few sentences. -->
+
+## Previews
+![PreviewOne](https://github.com/OB1Company/filehive/blob/master/web/screenshots/trending.jpg?raw=true)
+
+<!--Add some screenshots to give a preview of your product-->
+
+## Target Audience
+Anyone curating or maintaining datasets that is interested in funding or monetizing their work.
+
+<!--Describe who will be your project's users-->
+
+## Rough estimated user base (if applicable)
+TBD (pre-release)
+
+<!--How many users do you have right now?-->
+
+## Github repo
+https://github.com/OB1Company/filehive
+<!--Attach a link to your GitHub repo if it's OSS-->
+
+## Website
+https://filehive.app
+<!--Link your website if available-->
+
+## Docs
+https://github.com/OB1Company/filehive
+<!--Including a link to your project docs!-->
+
+## Team Info
+The OB1 team builds decentralized applications for people who appreciate them.
+<!-- Introduce your amazing team - how many team members are working on this project and who are they?-->
+
+### Team Size  
+5
+
+### Team members  
+Brian Hoffman
+Washington Sanchez
+Mike Wolf
+Chris Pacia
+Stevie Zollo
+
+## How the community can engage
```

Comment:
Could you please create a project discussion, with the category as Show and tell - Developers and title as [Category] Project Name(i.e: [Application] Slate), link the discussion in the How the community can engage section. This is to create a place to interact with the Filecoin community, share your updates and gain feedback on your projects!